src/scenarios/personalization/ProgressSet/ProgressSet.tsx

```diff
@@ -0,0 +1,181 @@
+  Divider,
+  Form,
+  Grid,
+  Header,
+  Label,
+  Segment,
+} from 'semantic-ui-react';
+import {
+  useScenarioHost,
+  VariableSearch,
+} from '@axinom/mosaic-fe-samples-host';
+import { getApolloClient } from '../../../apollo-client';
+import { SetProgressMutation } from './graphql-documents';
+
+export const ProgressSet: React.FC = () => {
+  const { activeProfile, logger } = useScenarioHost();
+  const [FrequencyOfProgressSaving, setFrequencyOfProgressSaving] =
+    useState<number>();
+  const [userAccessToken, setUserAccessToken] = useState<string>();
+  const [key, setKey] = useState<string>();
+  const [buttonState, setButtonState] = useState<boolean>(true);
+  const [time, setTime] = useState(0);
+  const [intervalId, setIntervalId] = useState<NodeJS.Timer>();
+
+  const apolloClient = getApolloClient(
+    new URL('graphql', activeProfile.personalizationServiceBaseURL).href,
+  );
+
+  useEffect(() => {
+    let interval: ReturnType<typeof setInterval> | undefined;
+    if (buttonState === false) {
+      interval = setInterval(() => {
+        setTime((prevTime) => prevTime + 10);
+      }, 10);
+    } else if (buttonState === true) {
+      clearInterval(interval);
+    }
+    return () => clearInterval(interval);
+  }, [buttonState]);
+
+  const setProgress = async (): Promise<void> => {
+    setButtonState(!buttonState);
+
+    if (buttonState === true) {
+      callMutationWithIntervale();
+    } else if (buttonState === false) {
+      clearInterval(intervalId);
+    }
+  };
+
+  function callMutationWithIntervale(): void {
+    const handler = setInterval(async () => {
+      // Call mutation
+      try {
+        const result = await apolloClient.mutate({
+          mutation: SetProgressMutation,
+          variables: {
+            input: {
+              key,
+              scope: 'PROFILE',
+              value: time,
+            },
+          },
+          context: {
+            headers: {
+              'Content-Type': 'application/json',
+              Authorization: `Bearer ${userAccessToken}`,
+            },
+          },
+          fetchPolicy: 'no-cache',
+        });
+
+        logger.log(`method [${setProgress.name}]`, 'output:', result.data);
+
+        if (result.errors) {
+          logger.error(result.errors);
+        }
+      } catch (error) {
+        if (error instanceof Error) {
+          logger.error(
+            `method [${setProgress.name}]`,
+            'output:',
+            error.message,
+          );
+        }
+      }
+    }, FrequencyOfProgressSaving);
+
+    setIntervalId(handler);
+  }
+
+  return (
+    <>
+      <Segment basic>
+        <Header size="huge">Progress</Header>
+        <Header size="small">
+          Required Services:
+          <Label>user-service</Label>
+          <Label>personalization-service</Label>
+        </Header>
+
+        <Divider />
+
+        <Container fluid>
+          <p>
+            This scenario demonstrates the main use case for saving the
```

Comment:
Would be good to check and update this description after updating the description for the get scenario